coralproject/talk
# [CORL-2364]: refactor to update all comment counts within moderate

## What does this PR do?

These updates make it so that `moderate` from the comments services also updates the counts when the status changes. Currently, there is a separate call to update the counts. With these changes, the call to update counts is included within `moderate`.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

 You can test this by seeing that the comment counts still update as expected when you approve a comment, reject a comment, delete a comment (when a user is deleted--you will first need to ensure this is allowed under `Configuration` --> `Authentication` --> `Commenter account management features` in the admin and then update when the user is scheduled to be deleted and when the job runs in `deleteScheduledAccounts`), and when archived comments are rejected by the rejector. See that everything else also still updates/works as expected.
 
## How do we deploy this PR?
